frontend/src/app/sessions/user-sessions-wrapper/create-sessions/create-readonly-session/create-readonly-session-dialog.component.html

```diff
@@ -56,11 +55,29 @@ <h2>Open read-only Session</h2>
         administrator or project lead.
       </div>
     }
+
+    @if (
+      maxNumberOfModels &&
+      this.selectedModelOptions().length > maxNumberOfModels
+    ) {
+      <div class="my-1 border p-2 text-sm shadow">
+        <p class="text-error">
+          The selected tool model only supports up to
+          {{ maxNumberOfModels }} session at a time.
+        </p>
+      </div>
+    }
```

Comment:
Can you create a story that covers the specific error case? Would be great :)

Reply:
I have created three stories that should cover the most interesting cases for now. If you miss a case, please leave a comment and I'll include it next week.